sample/wikifile.py

```diff
@@ -59,8 +59,42 @@ def get_duplicatefiles(self, dflimit="max", dfdir="ascending", dflocalonly=False
         self.payload['prop'] = None
         return df_list
 
+    def get_fileusage(self, fuprop="pageid|title|redirect", fulimit="max", funamespace="*", fushow="!redirect"):
+        prop = 'fileusage'
+        self.payload['prop'] = prop
```

Comment:
What does this do? Mention the fields? I saw `ns`, `pageid` and `title`, but I didn't see any `redirect`s.